src/main/kotlin/no/nav/syfo/api/model/SenOppfolgingKandidatResponseDTO.kt

```diff
@@ -42,13 +42,15 @@ fun SenOppfolgingKandidat.toResponseDTO(): SenOppfolgingKandidatResponseDTO = Se
             onskerOppfolging = it.onskerOppfolging,
         )
     },
-    vurderinger = this.vurderinger.map {
-        SenOppfolgingVurderingResponseDTO(
-            uuid = it.uuid,
-            begrunnelse = it.begrunnelse,
-            type = it.type,
-            veilederident = it.veilederident,
-            createdAt = it.createdAt.toLocalDateTime(),
+    vurderinger = this.vurdering?.let {
+        listOf(
+            SenOppfolgingVurderingResponseDTO(
+                uuid = it.uuid,
+                begrunnelse = it.begrunnelse,
+                type = it.type,
+                veilederident = it.veilederident,
+                createdAt = it.createdAt.toLocalDateTime(),
+            )
         )
-    }
+    } ?: emptyList()
```

Comment:
Vi venter litt med å endre DTOen, fordi det kan vi gjøre senere i en egen PR. Da slipper vi at denne oppgaven blir for stor mtp front end endringer osv.